changelogs/unreleased/gh-4693-formats-for-standalone-tuples.md

```diff
@@ -0,0 +1,7 @@
+## feature/box
```

Comment:
In commit message:
> Introduce `box.tuple.format` object, a Lua wrapper around tuple format:
    these objects own a tuple format, which is almost equivalent to
    space:format, except for check constraints and foreign key constraints
    being disabled (though they appear to be present for compatibility).

For compatibility with what? This is a new feature. It seems the only reason is that we don't want to properly validate and transform the received format into something more strict and small, and prefer to rely on the generic Lua->msgpack encoder. Same about `collation`, `nullable_action`, `constraint`, `foreign_key`, etc - these make absolutely no sense outside of a space and/or indexes.

I can see only one reason why this could be needed - to easily serialize the space format into an IProto response for tuples fetched directly from a space. But this could be fixed by serializing only the relevant format data into its binary form - field types and names. Full data in the binary representation is not needed for anything.

Not necessary to fix, adding it here for ordnung.

Reply:
> For compatibility with what?

With `space:format`.

Reply:
> ordnung

Ordnung?